docs/modules/deployment/pages/upgrade/disaster-recovery/restore-debian-ubuntu.adoc

```diff
@@ -0,0 +1,89 @@
+[[restore-debian-ubuntu]]
+= Restore OpenNMS {page-component-title} in Debian / Ubuntu installations
```

Comment:
```suggestion
= Restore OpenNMS {page-component-title} in Debian/Ubuntu installations
```
If we go for the tab structure for each OS as with the installation, we can remove the OS from the title.